package.json

```diff
@@ -23,13 +23,14 @@
   },
   "dependencies": {
     "boom": "4.x.x",
-    "hoek": "4.1.0"
+    "hoek": "4.1.0",
+    "joi": "9.1.0"
```

Comment:
Just to check, I assume you are with leaving https://github.com/johnbrett/hapi-auth-bearer-token/pull/101#discussion-diff-82131797R32 there?